vault/audit.go

```diff
@@ -294,7 +344,7 @@ type backendEntry struct {
 // AuditBroker is used to provide a single ingest interface to auditable
 // events given that multiple backends may be configured.
 type AuditBroker struct {
-	l        sync.RWMutex
+	sync.RWMutex
```

Comment:
For curiosity, what is the advantage of embedding `sync.RWMutex` instead of having a field?